src/datascience-ui/interactive-common/variableExplorer.tsx

```diff
@@ -178,15 +180,57 @@ export class VariableExplorer extends React.Component<IVariableExplorerProps, IV
         }
         if (
             prevState.containerHeight !== this.state.containerHeight ||
-            prevState.gridHeight !== this.state.gridHeight
+            prevState.gridHeight !== this.state.gridHeight && !this.props.viewMode
         ) {
             return true;
         }
 
+        // In view mode, we need to update when our height changes
+        if (
+            this.props.viewMode &&
+            (prevState.viewHeight !== nextProps.viewHeight)
+        ){
+            return true;
+        }
+
         return false;
     }
 
     public render() {
+        // This control renders differently when hosted in a versus a document
+        if (this.props.viewMode) {
```

Comment:
This is where we split the rendering. There were a number of differences, so I just used two different functions for the view mode versus document mode.

Reply:
I'm not sure which one is which? Is view mode inside of a webview?

Reply:
@rchiodo Yeah, in retrospect the name isn't the best, view is too general. I was thinking view versus document...maybe standaloneMode? That's really the crux of the issue here as the variable view is hosted all by itself outside of the document context.

Reply:
variableWebviewViewMode seems descriptive, but too wordy.

Reply:
standaloneMode gets my vote. That's clear to me at least.